machine.go

```diff
@@ -530,6 +533,28 @@ func (m *Machine) startInstance(ctx context.Context) error {
 	return err
 }
 
+// SetMetadata sets the machine's metadata for MDDS
+func (m *Machine) SetMetadata(ctx context.Context, metadata interface{}) error {
+	ctx, cancel := context.WithCancel(ctx)
+	defer cancel()
+
+	respcreated, respnocontent, err := m.client.PutMmds(ctx, metadata)
+
+	if err == nil {
+		var message string
+		if respcreated != nil {
+			message = respcreated.Error()
+		}
+		if respnocontent != nil {
+			message = respnocontent.Error()
+		}
+		m.logger.Printf("SetMetadata successful: %s", message)
+	} else {
+		m.logger.Printf("Error setting metadata: %s", err)
```

Comment:
This should be logged at Error severity, and drop "Error" from the message string.

Reply:
this pattern was too copied from `startInstance` - would you like me to change it as well?